src/Components/UploadPhoto/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC } from 'react';
+
+import { TouchableOpacity, Image, StyleSheet, Text } from 'react-native';
+
+import colors from '../../../utils/colors';
+
+type Upload = {
+  uri: string;
+  onPress: () => void;
+};
+
+const UploadPhoto: FC<Upload> = ({ uri, onPress }) => {
+  const initialIconPath =
+    'file:///Users/arturkhachatryan/Downloads/add-photo.png';
```

Comment:
use files from project not from computer.
and import local files outside from component